### PR TITLE
entry points - draft/craeted

### PR DESCRIPTION
**Pull request by planteater**
by me

### CODE REVIEW

File: freezer.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { SelectionDecorationProvider } from "./providers/selectionDecorationProv
 import { SetServerUrlRequestType} from "./protocols/agent/agent.protocol";
 // import { WebviewSidebarActivator } from "./views/webviewSidebarActivator";
 
+// entry point test
```
Comment (planteater) [inline freezer.js:21]: line 21 entry pt test comment

**Review by planteater — COMMENTED** [review]
meh

File: freezer.js (hunk 1 above)
Comment (planteater) [reply freezer.js]: reply to outdated comment

File: freezer.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ import { SelectionDecorationProvider } from "./providers/selectionDecorationProv
 import { SetServerUrlRequestType} from "./protocols/agent/agent.protocol";
 // import { WebviewSidebarActivator } from "./views/webviewSidebarActivator";
 
+// entry point test for the love of foo
```
Comment (bcanzanella) [inline freezer.js:21]: test

**Pull request closed without merging**